data.sql

```diff
@@ -1,5 +1,15 @@
-/* Populate database with sample data. */
```

Comment:
Please add the project title and description in README. Kindly make sure README is professional and descriptive.